docs/DeveloperGuide.md

```diff
@@ -283,33 +283,97 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+(For all use cases below, the **System** is the `UNion Application` and the **Actor** is the `user`, unless specified otherwise)
```

Comment:
capitalisation should be `UNIon`

examples of valid system names used in the textbook include `LearnSys` and `SquareGame`, so I think we can skip the contraction and just use `UNIon` as our system name